README.md

```diff
@@ -68,6 +68,10 @@ repositories:
 - name: insecure
    url: https://charts.my-insecure-domain.com
    caFile: optional_ca_crt
+# Advanced configuration: You can skip the verification of TLS for an https repo
+- name: skipTls
+    url: https://ss.my-insecure-domain.com
+    skipTlsVerify: true
```

Comment:
I prefer `skipTLSVerify` so that it is consistent with e.g. the existing tlsCACert. Let me do that change and merge this soon. Thank you for your patience and contribution!